package.json

```diff
@@ -137,16 +137,17 @@
     "markdown-to-jsx": "7.1.8",
     "nanoid": "^3.1.12",
     "node-polyglot": "^2.4.0",
-    "react": "^17.0.2",
-    "react-accessible-accordion": "^3.3.5",
+    "react": "18.2.0",
```

Comment:
Just double checking that the exact version flavor here is intentional

Reply:
I prefer to do this on big packages, especially since I want to make sure we pull the same version down in bloom. But it's mostly a personal preference, so I could go back to the `^` if the team prefers that way